### DISCUSSION TITLE
DirectX12 'ft2build.h' : No such file or directory

### DISCUSSION THREAD

**Discussion by tofflez** [ocornut/imgui#7796] [New Users Build/Link/Run issues ONLY!]
### Version/Branch of Dear ImGui:

Version 1.90.9 Branch: master

### Back-ends:

imgui_impl_win32.cpp + imgui_impl_dx12.cpp

### Compiler, OS:

Windows 11 + MSVC 2022

### Details:

**'ft2build.h' : No such file or directory**

Hi ! I searched everywhere i can to see how to fix this error but i can't find someone who explain how to install the freetype2 lib to avoid this error ! So im hoping having someone who can guide me here ! Im sorry if this sound like a dumb question, it's actually my first time using Dear ImGui 😅. Im compiling the DirectX12 Example. Im on Windows 11 using Visual Studio 2022 if this can help !
I didn't find how to include it in my project as i don't know if i already have freetype installed ^^'

### Screenshots/Video:

![image](https://github.com/user-attachments/assets/2696a43f-29e2-4170-a63d-b5f0c207a065)

**Comment by PathogenDavid** [comment]
There is some guidance on ways you might acquire FreeType in [the imgui_freetype readme](https://github.com/ocornut/imgui/tree/0e4dcfa55299eebd4659181b3622d24566f2f764/misc/freetype).

However do note that FreeType is not necessary to use Dear ImGui, it's an optional component.

Everything in the `misc` folder is optional. You probably don't want `binary_to_compressed_c.cpp` or `imugi_single_file.h` in your solution. You also probably shouldn't be adding the `main.cpp` from the example either.

The only things you *need* to add are the `*.h`/`*.cpp` files from the root of the repo and the `backends/imgui_impl_*` files for the backends you intend to use (that part you've done correctly.)

**Comment by Radagan** [comment]
In case it helps someone later on, I ran into this same issue on Linux. In my case, it was simply due forgetting to link with Freetype::Freetype.
